src/test/e2e/api/admin/api-token.auth.e2e.test.ts

```diff
@@ -74,7 +74,7 @@ test('editor users should only get client or frontend tokens', async () => {
         projects: [],
         tokenName: '',
         username: 'test',
-        secret: '1234',
+        secret: '*:1234',
```

Comment:
Let's be good citizens here - the token format was previously wrong because it should have been `project:environment.hash`, we've changed it to `project:hash` but let's go all the way to `project:environment.hash` so we're honest about what `secret` should hold